docs/tools/vdb_table/data/myscale.json

```diff
@@ -132,13 +132,13 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://clickhouse.com/docs/en/engines/table-engines/special/distributed",
     "comment": ""
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
+    "unlimited": true,
```

Comment:
Any mention of this in the docs? Would be useful to understand the tradeoffs involved for large docs, if there are any,.

Reply:
We don't have a doc for this, so I added a comment.